src/components/dropDown/DropDown.jsx

```diff
@@ -0,0 +1,57 @@
+import React, { useState, useEffect, useRef } from "react";
+
+const DropDown = ({ list, note, setNote, setDropdown }) => {
+  const node = useRef();
+
+  const changeHandler = (checked, value) => {
+    if (checked) {
+      if (!note.tags.includes(value))
+        setNote((prev) => ({ ...prev, tags: [...prev.tags, value] }));
+    } else {
+      setNote((prev) => ({
+        ...prev,
+        tags: prev.tags.filter((item) => item !== value),
+      }));
+    }
+  };
+
+  const [user, showText] = useState(false);
+  const clickOutside = (e) => {
+    if (node.current.contains(e.target)) {
+      return;
+    }
+    setDropdown(false);
+    showText(false);
+  };
+
+  useEffect(() => {
+    document.addEventListener("mousedown", clickOutside);
+    return () => {
+      document.removeEventListener("mousedown", clickOutside);
+    };
+  }, [user]);
```

Comment:
I did not completely understand the functionality of this useEffect. I guess it is to hide the dropdown when clicked anywhere on the page other than the dropdown container. I think you can try implementing the same functionality using onMouseDown event rather than using vanilla js. Even I need to check how to implement this but just a suggestion.

Reply:
I tried onMouseDown before. But it did not work.
Hence implemented it this way. There is also a react package called `react-onclickoutside`. But it gave some warnings.